solidus_related_products.gemspec

```diff
@@ -29,8 +29,8 @@ Gem::Specification.new do |spec|
   spec.executables = files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency 'solidus_core', '>= 2.0.0', '< 4'
-  spec.add_dependency 'solidus_backend', ['>= 2.0.0', '< 3']
+  spec.add_dependency 'solidus_core', '>= 2.0.0', '< 3.1'
+  spec.add_dependency 'solidus_backend', ['>= 2.0.0', '< 3.1']
```

Comment:
@vokshirg is there any reason for going only `< 3.1` instead of `< 4`?

Reply:
there is could be breaking changes 🤷‍♂️